leetcode.js

```diff
@@ -17,3 +17,96 @@ var fizzBuzz = function (n) {
 
 var test = fizzBuzz(15);
 console.log(test);
+
+// Minimum and Maximum Number in an Array
+
+function minMaxSum(arr) {
```

Comment:
please note you are adding all this code into the recipe site repository. Ideally you should create new repos for this or for separate learnings. Maybe create a learnings repo, and a leetcode repo separately - my suggestion